project.clj

```diff
@@ -40,7 +41,11 @@
   ;; code that we have.
   :classifiers [["test" :testutils]]
 
-  :profiles {:dev {:source-paths ["examples/shutdown_app/src"
+  :profiles {:cljfmt {:plugins [[lein-cljfmt "0.3.0"]
```

Comment:
i believe 0.3.0 was the newest when any of those project integrated this, 0.4.0 is less than a month old